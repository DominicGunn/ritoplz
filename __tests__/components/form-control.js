'use strict'

/* eslint-env jest */

import renderer from 'react-test-renderer'

import FormControl from '../../components/ui/form-control'

describe('ui form-control component', () => {
  describe('snapshots', () => {
    it('should render form-control', () => {
      const tree = renderer.create(<FormControl />).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
