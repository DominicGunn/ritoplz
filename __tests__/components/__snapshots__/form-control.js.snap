// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui form-control component snapshots should render form-control 1`] = `
<div
  className="form-control"
  data-jsx={2101790025}
>
  <a
    className="link default"
    data-jsx={1642471407}
    href="/profile"
    onClick={[Function]}
    style={undefined}
  >
    Cancel
  </a>
  <button
    className="success"
    data-jsx={2023613154}
    disabled={false}
    onClick={undefined}
    style={undefined}
    type="submit"
  >
    Save settings
  </button>
</div>
`;
